src/ngraph/op/util/attr_types.cpp

```diff
@@ -126,4 +128,17 @@ namespace ngraph
     {
         return s << as_string(type);
     }
+
+    op::AutoBroadcastType op::AutoBroadcastSpec::type_from_string(const std::string& type) const
+    {
+        static const std::map<std::string, AutoBroadcastType> allowed_values = {
+            {"none", AutoBroadcastType::NONE},
```

Comment:
Let's keep the strings and enum values consistent.

Reply:
Done